e2e-tests/playwright/tests/functional/channels/edit_file_attachments/edit_file_attachment.spec.ts

```diff
@@ -0,0 +1,372 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {test} from '@e2e-support/test_fixture';
+import pages from '@e2e-support/ui/pages';
+import {expect} from '@playwright/test';
+import {duration, wait} from '@e2e-support/util';
+
+test('MM-T5654_1 should be able to edit post message', async ({pw}) => {
```

Comment:
There are 4 automated tests here for MM-T5654 (_1 / _2 / _1 / _3). However, looking at the [test case](https://github.com/mattermost/mattermost-test-management/blob/main/data/test-cases/channels/edited-post/MM-T5654.md#scenario-1-user-should-be-able-to-add-attachments-while-editing-a-post), there's only one step (with 3 scenarios). How about update the written test case to either put it in separate "steps" or different test cases.